tests/test_dependency_rendering.py

```diff
@@ -9,35 +9,29 @@
 
 
 class SimpleComponentAlternate(component.Component):
-    def context(self, variable):
-        return {}
-
-    def template(self, context):
-        return "simple_template.html"
+    template_name = "simple_template.html"
 
     class Media:
         css = "style2.css"
         js = "script2.js"
 
 
 class SimpleComponentWithSharedDependency(component.Component):
+    template_name = "simple_template.html"
+
     def context(self, variable, variable2="default"):
```

Comment:
This should change to get_context.

Reply:
Would you mind checking so that there are no "def context" left after this?

Reply:
Missed that somehow, will check again.